### PR TITLE
Explicit paths for modules

### PR DESCRIPTION
**Pull request by smoe**
On https://docs.python.org/3/tutorial/modules.html it is said that relative paths would be possible. Now it is a bit Java-like with explicit full paths. I think I like that better - at least it was easier.

This was done in a semi-automated manner and I may have missed a few. Will address packaging of the mirtop module now and then continue.

### CODE REVIEW

File: test/test_cluster.py [hunk 1]
```diff
@@ -4,8 +4,8 @@
 import os.path as op
 import inspect
 import pybedtools
-import seqcluster.libs.parameters
-import seqcluster.libs.logger as mylog
+import seqcluster.seqcluster.libs.parameters
```
Comment (lpantano) [inline test/test_cluster.py]: is this right? seqcluster twice, or it should be just seqcluster.libs.parameters?

it is failing in python 2.7 right now.

**Comment by lpantano** [comment]
Thanks for doing this. I see travis is failing because `seqcluster.seqcluster.libs...` is not good for python 2.7, is this working on python 3?

**Comment by smoe** [comment]
Er, no. This is just a stupid hickup of a script I had written and I had though to have corrected them all. I'll check again and update. Sorry.

**Comment by smoe** [comment]
Apparently I had not checked in the test directory. Fixed.

**Comment by lpantano** [comment]
Thanks!

**Pull request merged**